File: fieldsync-inventory-backend/src/main/java/com/example/fieldsync_inventory_backend/dto/inventory/seed_batch/SeedBatchRequestDTO.java
package com.example.fieldsync_inventory_backend.dto.inventory.seed_batch;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class SeedBatchRequestDTO {
    private Long id;
    private Integer varietyId;
    private Integer generationId;
    private Integer seasonId;
    private Integer year;
    private Boolean grading;
    private Boolean germination;
    private String description;
}

File: fieldsync-inventory-app/app/src/main/java/com/example/fieldsync_inventory_app/data/remote/dto/seed_batch/SeedBatchRequestDto.kt
package com.example.fieldsync_inventory_app.data.remote.dto.seed_batch

data class SeedBatchRequestDto(
    val varietyId: Int,
    val generationId: Int,
    val seasonId: Int,
    val year: Int,
    val grading: Boolean,
    val germination: Boolean,
    val description: String?
)